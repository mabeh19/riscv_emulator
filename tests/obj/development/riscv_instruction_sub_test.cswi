20231102203344
-c
-x
ada
-gnatA
-Og
-ffunction-sections
-fdata-sections
-g
-gnatw.X
-gnatVa
-gnaty3
-gnatya
-gnatyA
-gnatyB
-gnaty-d
-gnatye
-gnatyI
-gnat2022
