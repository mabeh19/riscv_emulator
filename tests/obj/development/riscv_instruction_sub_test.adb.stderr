riscv_instruction_sub_test.adb:82:42: warning: negative literal of modular type is in fact positive [-gnatw.m]
riscv_instruction_sub_test.adb:82:42: warning: do you really mean 4294966646 when writing -650 ? [-gnatw.m]
riscv_instruction_sub_test.adb:82:42: warning: if you do, use qualification to avoid this warning [-gnatw.m]
